docs/pages/Components/RadioGroup.md

```diff
@@ -0,0 +1,70 @@
+| ------------------ | ----------------------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| content            | string                        |               | The text string for the option.                                                                                                                                                                                                                                          |
+| buttonKey          | string                        |               | A unique key-identifier for each option.                                                                                                                                                                                                                                 |
+| disabled           | boolean                       |               | Whether or not the radio button is selectable.                                                                                                                                                                                                                           |
+| ariaPosInSet       | number                        |               | Defines the current radio button's position in the radio group for accessibility purposes. It's recommended to set this value if radio buttons are not direct children of radio group. This value is auto-generated if radio buttons are direct children of radio group. |
+| ariaSetSize        | number                        |               | Defines the number of radio buttons in the group for accessibility purposes.It's recommended to set this value if radio buttons are not direct children of radio group. This value is auto-generated if radio buttons are direct children of radio group.                |
+| accessibilityLabel | string                        |               | An accessibility label for screen readers. If not provided, it will be set to the label of the radio button's content.                                                                                                                                                   |
+| componentRef       | `React.RefObject<IFocusable>` |               | A RefObject to access the IFocusable interface. Use this to access the public methods and properties of the component.                                                                                                                                                   |
+
+## Tokens:
+
+RadioButton supports the following tokens:
+
+1. borderColor – This changes the border color of the RadioButton.
```

Comment:
Curious why some of these are lists and others have tables for their tokens?

Reply:
Older format. Going to change it so it's all the same.

And on that note... Which do you think is better? List or table?